stylesheet.css

```diff
@@ -0,0 +1,45 @@
+#main {
+  background-color: rgb(238, 238, 238);
+  border-radius: 2px;
+}
+
+body {
+  text-align: center;
+  font-size: 100%;
+  font-family: apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Helvetica Neue', Arial, sans-serif;
+}
+
+ul {
+  max-width: 550px;
+  margin: auto;
+  text-align: left;
+}
+
+#image {
+  margin: auto;
+  font-size: 100%;
```

Comment:
Is there a reason why `font-size` is used for an image element?

Let's remove this redundant code.